src/Workleap.OpenApi.MSBuild/SpectralManager.cs

```diff
@@ -86,7 +84,7 @@ private async Task GenerateSpectralReport(string spectralExecutePath, string swa
             await this.AssignExecutePermission(spectralExecutePath, cancellationToken);
         }
 
-        var exitCode = await this._processWrapper.RunProcessAsync(spectralExecutePath, new[] { "lint", swaggerDocumentPath, "--ruleset", rulesetUrl, "--format", "html", "--output.html", htmlReportPath }, cancellationToken);
+        var exitCode = await this._processWrapper.RunProcessAsync(spectralExecutePath, new[] { "lint", swaggerDocumentPath, "--ruleset", rulesetUrl, "--format", "html", "--format", "pretty", "--output.html", htmlReportPath, "--fail-severity=warn" }, cancellationToken: cancellationToken);
```

Comment:
(same nit-pick comment about CancellationToken): why specifying the parameter name? Seems to add verbosity

Reply:
(also applicable in the couple others places)